zproject/dev_settings.py

```diff
@@ -187,9 +188,9 @@
 MEMCACHED_USERNAME: Optional[str] = None
 
 SCIM_CONFIG = {
-    "zulip": {
-        "bearer_token": "token1234",
-        "scim_client_name": "test-scim-client",
-        "name_formatted_included": True,
-    }
+    "zulip": SCIMConfigDict(
```

Comment:
For settings, I’d slightly prefer to keep the `{…}` syntax and annotate `SCIM_CONFIG: Dict[str, SCIMConfigDict]`, similar to `SOCIAL_AUTH_SAML_ENABLED_IDPS`. This makes it clearer that users can use the `{…}` syntax without adding explicit references to the `TypedDict`.